content/docs/guides/pipelines/pipelines-metrics.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Pipeline metrics"
+description = "Export and visualize pipeline metrics."
+weight = 3
+toc = true
+
+[menu.docs]
+  parent = "pipelines"
+  weight = 6
++++
+This page is for component author to export metrics from the component. For details about how to build your own component, please see [Editing Build Your Own Component](/docs/guides/pipelines/build-component).
```

Comment:
The title of the page is actually "Build your own component". A good tech writing tip is to use descriptive words rather than page titles in link text. This helps to avoid misquoting the title, and to future-proof us against changing titles, and to make the sentence flow better. Suggested code:

```
This page shows component authors how to export metrics from the component. For details about how to build a component, see the guide to [building your own component](/docs/guides/pipelines/build-component).
```

Reply:
done